Makefile

```diff
@@ -21,7 +22,7 @@ include ./deps/eng/tools/mk/Makefile.rust.defs
 
 export STAMP
 
-BUILD_PLATFORM = 20210826T002459Z
+BUILD_PLATFORM :=	20240111T002438Z
```

Comment:
That happens to be the PI of where the one zone landed.  Did you pick that for that specific reason? Or is there another?